README.md

```diff
@@ -1,2 +1,10 @@
 # frequenz-copier-rs
+
+[![Copier](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/copier-org/copier/master/img/badge/badge-grayscale-inverted-border-orange.json)](https://github.com/copier-org/copier)
+
 An opinionated Copier template to create Frequenz rust projects
+
+## How to copy this template
+
+1. Install [Copier](https://copier.readthedocs.io/en/stable/#installation)
+2. Run `copier copy https://github.com/frequenz-floss/frequenz-copier-rs.git path/to/destination`
```

Comment:
Maybe add a section on updating the template?

Also how does copier knows this file should not be part of the template (copied to the new repos), only because you also have a `README.md.jinja`?

Reply:
Yes, it is overwritten by the template. That's a nice feature for us, because we can have a README that only applies to users of the template but is gone when having it generated.

Reply:
What we did with Python is to generate the project repo too, and all the templates go in there, everything that's not in there, is just part of the template repo itself (like migration script, or *hooks*). I never liked the templates in the file names, but it is actually pretty nice to me to have a clear boundary between the template repo itself and the generated project. If it is possible to make this distinction better in copier too, like having the templates in a sub-directory, I think it might be worth doing.